src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -50,14 +75,42 @@ export const ContactPolicy = InjectAppServices(
           setFormSubmitted(true);
           resetForm({ values });
           setSettings(values);
+          setError(false);
         } else {
-          console.log('Error updating account settings');
+          setError(true);
         }
       } finally {
         setSubmitting(false);
       }
     };
 
+    const validate = (values) => {
+      setError(false);
+      setFormSubmitted(false);
+      const errors = {};
+
+      const amountIsEmpty = values.emailsAmountByInterval === '';
+      const intervalIsEmpty = values.intervalInDays === '';
+
+      if (amountIsEmpty || intervalIsEmpty) {
```

Comment:
Formik handles 2 types of validations: at the form-level and at the field-level. In case of form-level validations, you can pass a validation function to it. You don't need to handle these validations manually. 
doc: https://formik.org/docs/guides/validation